### PR TITLE
Let's see what this does with netlify's autodeploy

### PR DESCRIPTION
**Pull request by caronl**

### CODE REVIEW

**Comment by caronl** [comment]
After last commit : Based on sharing netlify's autodeploy on LinkedIn, the URL seems to exhibit the intended behavior. However, for FB it looks like we still need to update the [thumbnail debugger](https://developers.facebook.com/docs/sharing/best-practices/#debug) by hand.

**Comment by caronl** [comment]
For FB to work, alternatives are (in increasing order of complexity, but decreasing order of cleanliness) :
1) We hardcode the logo in the og metadata
    - We might want to wait for the official logo release for that
    - Instead of ```{{.Params.featuredpath}}/{{.Params.featured}}```, my commit + would have been ```img/main/logo.jpg```
2) We enforce mandatory size and file extension for featured images
    - This will be a pain in the ass to manage, and I don't think it respects our intent of easy article deployment
3) We find how to programatically find the extension, width and height of the featured image to put in the og metadata
    - Haven't been successful so far

**Comment by caronl** [comment]
Also, someday, we might want to improve other tags as described [here](http://ogp.me/#metadata)

**Comment by davebulaval** [comment]
Still not working for me. When you use the share to linkedIn from the blog post.

**Comment by davebulaval** [comment]
Facebook is working fine.

**Comment by caronl** [comment]
@davebulaval as-tu essayé de share le blog (master, qui n'est pas à jour avec cette branche) ou l'auto-deploy de netlify (cette branche)?

**Comment by caronl** [comment]
Et Facebook fonctionne parce qu'on a debuggé à la main déjà pour le master et l'auto-deploy. Probablement qu'un petit commit insignifiant juste pour faire changer l'adresse de l'auto-deploy te permettrait de tester LinkedIn et FB from scratch.

**Comment by davebulaval** [comment]
@caronl ...oups

**Comment by AnnieDeshaies** [comment]
@caronl Le problème avec Facebook et LinkedIn est dû au fait qu'on l'utilise en localhost. En release, ça va s'arranger. 

Ça serait nice de merger cette PR qui date de 2018 @AntoineButeau @davebulaval @stecaron ?

**Comment by AnnieDeshaies** [comment]
Facebook utilise les _meta_ `og:[...]` pour afficher ses informations en sharing. Twitter utilise les _meta_ `twitter:[...]`. 

En ce moment, le `og:image` pointe vers une adresse qui retourne un 404 et contient des _slash_ de trop.
`<meta property="og:image" content="https://dotlayer.org//images/logo.png">`

J'ai essayé un fix, on peut l'essayer 🤙🤙🤙

File: config.toml [hunk 1]
```diff
@@ -25,6 +25,8 @@ enableEmoji   = true
   navbarTitle          = ".layer"
   # Sets where "View More Posts" links
   viewMorePostLink     = "/blog/"
+  # Sets default img for thumbnails
+  og_image             = "img/headers/hackathon-code-cover.JPG"
```
[the remark this answers is not in the document]
Comment (davebulaval) [reply config.toml]: I like this default choice.

**Pull request merged**